scripts/install/lidarr.sh

```diff
@@ -48,15 +51,15 @@ Type=simple
 User=${user}
 Group=${user}
 Environment="TMPDIR=/home/${user}/.tmp"
-ExecStart=/usr/bin/mono /home/${user}/Lidarr/Lidarr.exe -nobrowser
+ExecStart=/usr/bin/mono /opt/Lidarr/Lidarr.exe -nobrowser
 ExecStop=-/bin/kill -HUP
```

Comment:
We should maybe remove this line -- it's not in the wiki for systemd

https://github.com/lidarr/Lidarr/wiki/Autostart-on-Linux

Reply:
Lidarr 0.8 will be coming out rather soon and is .net core so mono will be dropped FYI

Also all 4 *arrs are now over on the Servarr wiki; the GitHub wikis are depreciated.

app and thus start up should be in opt

Reply:
> Also all 4 *arrs are now over on the Servarr wiki; the GitHub wikis are depreciated.

Is there a specific reference for this in this PR? this line reference doesn't seem to be it :p

> Lidarr 0.8 will be coming out rather soon and is .net core so mono will be dropped FYI

I'll believe it when I see it hahahah! Either way if you'd like to help out with that, it would be appreciated. When we did that for Radarr, it was done via #482, #486 and #487. I suppose it's essentially the same changes.